packages/react-components/react-menu/src/components/MenuTrigger/useMenuTrigger.styles.ts

```diff
@@ -31,6 +31,7 @@ export const useMenuTrigger_unstable = (props: MenuTriggerProps): MenuTriggerSta
   const triggerId = useMenuContext_unstable(context => context.triggerId);
   const openOnHover = useMenuContext_unstable(context => context.openOnHover);
   const openOnContext = useMenuContext_unstable(context => context.openOnContext);
```

Comment:
Can you please also remove `.styles.` suffix from this file? Looks that I renamed it by mistake